Sources/NIOWebSocketClient/main.swift

```diff
@@ -208,6 +190,33 @@ default:
     connectTarget = .ip(host: defaultHost, port: defaultPort)
 }
 
+let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+let bootstrap = ClientBootstrap(group: group)
+    // Enable SO_REUSEADDR.
+    .channelOption(ChannelOptions.socketOption(.so_reuseaddr), value: 1)
+    .channelInitializer { channel in
+
+        let httpHandler = HTTPInitialRequestHandler()
```

Comment:
I think the `init` needs to be updated here to actually take the target.

Reply:
Ah yes, that got lost when making the commits. Fixed now.